packages/template-retail-react-app/app/components/list-menu/index.jsx

```diff
@@ -219,15 +218,16 @@ ListMenuPopover.propTypes = {
  * users use the keyboard Tab key to focus over the chevron icon and press Enter.
  *
  * @param maxColumns The maximum number of columns that we want to use per row inside the ListMenu.
+ * @param root
  */
-const ListMenu = ({maxColumns = MAXIMUM_NUMBER_COLUMNS}) => {
-    const {root, itemsKey} = useCategories()
+const ListMenu = ({root, maxColumns = MAXIMUM_NUMBER_COLUMNS}) => {
+    const itemsKey = 'categories'
     const theme = useTheme()
     const {baseStyle} = theme.components.ListMenu
-    const [ariaBusy, setAriaBusy] = useState('true')
+    const [ariaBusy, setAriaBusy] = useState(true)
```

Comment:
I think you may have forgotten to update line 238 (?)

```diff
- aria-busy={ariaBusy}
+ aria-busy={ariaBusy ? "true" : "false"}
```

Reply:
We don't need too. React will turn it into string when rendering
![image](https://user-images.githubusercontent.com/52219283/228974719-e95068b4-c1f9-497e-8709-3c5807a100f7.png)